src/applications/vaos/new-appointment/index.jsx

```diff
@@ -143,14 +148,6 @@ function NewAppointmentSection({
   );
 }
 
-function mapStateToProps(state) {
-  return {
-    isCernerOnlyPatient: selectIsCernerOnlyPatient(state),
-    flatFacilityPageEnabled: selectUseFlatFacilityPage(state),
-    providerSelectionEnabled: selectUseProviderSelection(state),
-  };
-}
-
-export const NewAppointment = connect(mapStateToProps)(NewAppointmentSection);
+export const NewAppointment = NewAppointmentSection;
```

Comment:
You could probably rename the function above to `NewAppointment` and export it, instead of having this extra line.

Reply:
Ok, that was my first inclination but thought maybe outside of scope 👍